docker/mysql/db/db_upgrade/portal/Update_RMC_DB_2023082401-2023083000.sql

```diff
@@ -0,0 +1,69 @@
+START TRANSACTION;
+
+CREATE TABLE `destruction_log` (
+    recordID MEDIUMINT(8) UNSIGNED NOT NULL,
+    categoryID VARCHAR(20) NOT NULL,
+    destructionTime INT(10) UNSIGNED DEFAULT 0 NULL,
+    PRIMARY KEY (recordID)
+);
+CREATE INDEX destructionTime
+    ON `destruction_log` (`destructionTime`);
+CREATE INDEX destructionForm
+    ON `destruction_log` (`categoryID`);
+
+ALTER TABLE `categories` MODIFY `destructionAge` mediumint(8) UNSIGNED NULL DEFAULT NULL;
```

Comment:
should this be an int?

Reply:
This field is representing destruction time in days - I'll touch in with Pete about whether it might be better to have it as a timestamp.